get_started/what_is_citus.rst

```diff
@@ -1,16 +1,27 @@
 .. _what_is_citus:
 
-What is Citus?
-==============
+Why Citus?
+==========
 
-Citus horizontally scales PostgreSQL across multiple machines using sharding and replication. Its query engine parallelizes incoming SQL queries across these servers to enable human real-time (less than a second) responses on large datasets.
+Fast-growing multi-tenant apps want to add new customers, deliver great performance, and not have to worry about database infrastructure. Data analysts want sub-second response times for customer-facing analytics dashboards, even with real-time ingestion, very large data sets, complex queries, and lots of concurrent users.
```

Comment:
Can we add more links, maybe to the website? E.g. the Product or Use-case pages?